ecommerce/models.py

```diff
@@ -46,6 +50,43 @@ def __str__(self):
         """Description for Order"""
         return "Order {}, status={} for user={}".format(self.id, self.status, self.user)
 
+    def to_dict(self):
+        """
+        Get a serialized representation of the Order and any attached Lines
+        """
+        data = serialize_model(self)
+        data['lines'] = [serialize_model(line) for line in self.line_set.all()]
+        return data
```

Comment:
fair enough. as long as we have a somewhat consistent API for the serialization, this approach should be fine